pkg/config/services.go

```diff
@@ -473,6 +473,16 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile("^(?P<OutputARN>.*:output:.*)$"),
 		},
 	},
+	{
+		Namespace: "AWS/MediaConvert",
+		Alias:     "mediaconvert",
+		ResourceFilters: []*string{
+			aws.String("mediaconvert"),
+		},
+		DimensionRegexps: []*regexp.Regexp{
+			regexp.MustCompile("^(?P<QueueARN>.*:queue:.*)$"),
+		},
+	},
```

Comment:
Is this regex used to created metrics per Queue dimensions right?

Reply:
The regex is used to correctly extract dimensions from metrics discovered in CloudWatch, so that the metric can be associated to the correct resource ARN. In this case, if you run the `aws cloudwatch list-metrics` cli you should see metrics with a QueueARN dimension. Is this the case? (can you paste the output of the cli command if you have it please?)

Reply:
I'm seeing this:

`  {
            "Namespace": "AWS/MediaConvert",
            "MetricName": "LongestBlackSegmentDetected",
            "Dimensions": [
                {
                    "Name": "Queue",
                    "Value": "arn:aws:mediaconvert:eu-west-1:638618404227:queues/vtmgo"
                }
            ]
        },`

Reply:
So I updated it to Queue and the expression as well to take the full QueueArn

Reply:
`
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:jobs/1683887580939-x51l9b",tag_Application="",tag_Selector="",tag_Squad="",tag_Stage="",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:jobs/1683887585192-po9c4c",tag_Application="",tag_Selector="",tag_Squad="",tag_Stage="",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:jobs/1683887594813-lc5n40",tag_Application="",tag_Selector="",tag_Squad="",tag_Stage="",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:jobs/1683887599348-s2o3q6",tag_Application="",tag_Selector="",tag_Squad="",tag_Stage="",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:jobs/1683887732466-g9gw95",tag_Application="",tag_Selector="",tag_Squad="",tag_Stage="",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/Default",tag_Application="video-upload-service",tag_Selector="",tag_Squad="DeVideotheek",tag_Stage="prod",tag_map_migrated=""} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/streamz",tag_Application="video-upload-service",tag_Selector="Streamz",tag_Squad="DeVideotheek",tag_Stage="prod",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-1",tag_Application="video-upload-service",tag_Selector="",tag_Squad="DeVideotheek",tag_Stage="prod",tag_map_migrated=""} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-2",tag_Application="video-upload-service",tag_Selector="",tag_Squad="DeVideotheek",tag_Stage="prod",tag_map_migrated=""} 0
aws_media_convert_info{name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/vtmgo",tag_Application="video-upload-service",tag_Selector="VTM GO",tag_Squad="DeVideotheek",tag_Stage="prod",tag_map_migrated="d-server-02z4b03pu7mev1"} 0
# HELP aws_media_convert_jobs_completed_count_sum Help is not implemented yet.
# TYPE aws_media_convert_jobs_completed_count_sum gauge
aws_media_convert_jobs_completed_count_sum{account_id="638618404227",dimension_Queue="arn:aws:mediaconvert:eu-west-1:638618404227:queues/Default",name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/Default",region="eu-west-1"} NaN
aws_media_convert_jobs_completed_count_sum{account_id="638618404227",dimension_Queue="arn:aws:mediaconvert:eu-west-1:638618404227:queues/streamz",name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/streamz",region="eu-west-1"} 2
aws_media_convert_jobs_completed_count_sum{account_id="638618404227",dimension_Queue="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-1",name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-1",region="eu-west-1"} 1
aws_media_convert_jobs_completed_count_sum{account_id="638618404227",dimension_Queue="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-2",name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/video-upload-service-reserved-2",region="eu-west-1"} NaN
aws_media_convert_jobs_completed_count_sum{account_id="638618404227",dimension_Queue="arn:aws:mediaconvert:eu-west-1:638618404227:queues/vtmgo",name="arn:aws:mediaconvert:eu-west-1:638618404227:queues/vtmgo",region="eu-west-1"} NaN
# HELP yace_cloudwatch_apigatewayapi_requests_total 
`

Reply:
I removed the resourcefilters for now for mediaconvert. As you can see, for now only jobs are tagged, but is this useful? Since i don't see any metrics directly related to the jobs, more to the queues itself.

Reply:
Thanks for testing this. ResourceFilters should always be specified, so that tagged resources are captured correctly.

Thanks also for changing the regex, can you please just make it a bit more specific than a `.*`?